app/helpers/dashboards_helper.rb

```diff
@@ -8,15 +8,15 @@ def date_range(date: Time.zone.now)
     end
   end
 
-  def month_range(date: date)
+  def month_range(date: Time.zone.now)
```

Comment:
Is this change intentional?

Reply:
This was an existing bug that became breaking in the 2.7 upgrade!

Reply:
Damn kwargs 🤣 
![image](https://user-images.githubusercontent.com/31595784/99009502-1ebad100-2516-11eb-9918-d185374935c5.png)